lnrpc/routerrpc/router_backend.go

```diff
@@ -1370,6 +1370,9 @@ func convertPaymentStatus(dbStatus channeldb.PaymentStatus) (
 	lnrpc.Payment_PaymentStatus, error) {
 
 	switch dbStatus {
+	case channeldb.StatusInitiated:
+		return lnrpc.Payment_INITIATED, nil
```

Comment:
Am I right that previously we'd send an `Payment_IN_FLIGHT` update when payments were created (but no HTLCs), and now we send this new state?

Reply:
Yeah exactly.

Reply:
Trying to decide whether that's a breaking  API change 🤔  probably fine?

Reply:
Good question. Checked the `TrackPaymentV2` and `TrackPayments`, because we don't do `notifySubscribers` inside `InitPayment` in control tower, this status is not sent, so from the outside, the events received from the stream should stay the same. Meanwhile I think we should notify it in `InitPayment`, which will then break the API a bit, probably in another PR.